cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/nosql/NoSqlStructuredTable.java

```diff
@@ -479,17 +566,70 @@ protected StructuredRow computeNext() {
 
     @Override
     public void close() {
-      closeScanner();
+      currentScanner.close();
```

Comment:
this one should still be closeScanner(), the previous comment was only for the FilterByIterator, where there is just one iterator.

Reply:
yes this was an accidentally changed  too